shared/chat/inbox/row/small-team/index.js

```diff
@@ -58,7 +58,8 @@ class SmallTeam extends React.PureComponent<Props, State> {
 
   _onMouseLeave = () => this.setState({isHovered: false})
   _onMouseOver = () => this.setState({isHovered: true})
-  _onShowMenu = (showMenu: boolean) => this.setState({showMenu})
+  _onForceShowMenu = () => this.settate({showMenu: true})
```

Comment:
use property and not inline closure

Reply:
s/this.settate/this.setState, here and on the line below.